src/mixins/filing-template-mixin.ts

```diff
@@ -493,6 +493,11 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
         }
       }) || []
     )
+    // Store business contact info
+    this.setBusinessContact({
+      ...entitySnapshot.authInfo.contact
+    })
+
```

Comment:
As with alterations and change filings, this information will always be up to date. 
Regardless if its a new change filing or from a draft change filing, once the changes are saved, they take effect immediately.